src/cogent3/maths/stats/special.py

```diff
@@ -2,8 +2,12 @@
 (c) Stephen L. Moshier 1984, 1995.
 """
 
+import scipy
```

Comment:
Delete this import. The point here is not to replace our function with that from Scipy; rather, just use the decorator and tell users where to go to find the replacement.